installer/PowerToysSetup/Product.wxs

```diff
@@ -268,6 +268,7 @@
           <!-- NB: Needed since it's only referenced in XAML. -->
           <netfx:NativeImage Id="Interactivity" Platform="all" Priority="0"/>
         </File>
+        <File Source="$(var.BinX64Dir)\modules\Newtonsoft.Json.dll" />
```

Comment:
Is this required? This for `MSI` right

Reply:
Yes, this is for MSI. MSI will be deprecated but since I had already added ImageResizer to the MSI installer earlier I added this dll as well as the ImageResizer.exe won't launch unless it is present.